File: Bubble_Sort/Bubble_Sort.php
<?php

 $arr = array(2, 4, 3, 1, 6, 8, 4);
 //Bubble sort function
 function Bubble_Sort($arr, $size)
 {
 	for ( $i = 0; $i < $size - 1; $i++)
    {
        for ( $j = 0; $j < $size - $i - 1; $j++)
        {
            // Do swapp$ing
            if ($arr[$j] > $arr[$j + 1])
            {
                $temp = $arr[$j];
                $arr[$j] = $arr[$j + 1];
                $arr[$j + 1] = $temp;
            }
        }
    }

   for ( $k=0 ; $k<$size;$k++)
         echo "$arr[$k] ";
}
 // Bubble sort function calling
 Bubble_Sort($arr, 7);

 // Output
 // 1 2 3 4 4 6 8
 ?>

File: Bubble_Sort/Bubble_Sort.go
package main

import "fmt"

// function for Bubble Sort
func BubbleSort(array[] int)[]int {
	for i:=0; i< len(array)-1; i++ {
		for j:=0; j < len(array)-i-1; j++ {
			if (array[j] > array[j+1]) {
				array[j], array[j+1] = array[j+1], array[j]
			}
		}
	}
	return array
}

// Driver Function
func main() {
	array:= []int{2, 4, 3, 1, 6, 8, 4};
	fmt.Println(BubbleSort(array))
}
